web-client/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Doing `yarn workspace web-client create vite` will creates another directory inside of `web-client`. This is really ugly but I don't have any good idea to craete vite project safely. So I just moved everything outisde from `vite-project` (which is default created directory) and do some extraction for `tsconfig.json`.